### ISSUE TITLE
OpenBMC:Witherspoon:FVT- BMC factory reset not working

### ISSUE THREAD

**Issue by cdeadmin** [openbmc/openbmc#2326]
<cde:info> SW402195 </cde:info>

**Comment by anoo1** [comment]
```
BMC factory reset is not working on GHE.
It is not erasing or deleting vloumes and perstance files created by BMC.

Version Info:
ID="openbmc-phosphor"
NAME="Phosphor OpenBMC (Phosphor OpenBMC Project Reference Distro)"
VERSION="ibm-v1.99.9"
VERSION_ID="ibm-v1.99.9-0-r6-0-ga497e4e"
PRETTY_NAME="Phosphor OpenBMC (Phosphor OpenBMC Project Reference Distro) ibm-v1.99.9"
BUILD_ID="ibm-v1.99.9-0-r6"
```

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/6760> fstab: require service file to mount /var - fixes BMC factory reset
Resolves  #2326 OpenBMC:Witherspoon:FVT- BMC factory reset not working
